examples/website/geojson/app.js

```diff
@@ -1,7 +1,13 @@
 import React, {Component} from 'react';
 import {render} from 'react-dom';
 import {StaticMap} from 'react-map-gl';
-import DeckGL, {GeoJsonLayer} from 'deck.gl';
+import DeckGL, {
+  GeoJsonLayer,
+  AmbientLight,
```

Comment:
From an API audit perspective: 

* it could be worthwhile thinking about whether the new classes (AmbientLight, DirLight, PhongMaterial, LightingEffect etc) should be part of deck.gl core, or perhaps an optional `@deck.gl/effects` library. 

* It would be good to have a plan for how to support PBR (Physically based) rendering in parallel with/as an option to this system. I suspect the "light source" classes could be the same, just different materials / effect, but lets discuss.

* A lot of lighting functionality is general and could be part of luma.gl and just imported here.

Reply:
Right now luma.gl has its own lighting module and it's not being used by deck. Does it make sense moving these classes into either luma.gl core or a luma.gl submodule, and then only handle the deck-specific logic (mostly projection) here?